src/main/java/org/reaktivity/nukleus/http2/internal/Http2Connection.java

```diff
@@ -1529,6 +1530,19 @@ void mapPushPromise(
         httpHeaders.forEach(h -> builder.header(b -> mapHeader(h, b)));
     }
 
+    void mapTrailers(
+            ListFW<HttpHeaderFW> httpHeaders,
+            HpackHeaderBlockFW.Builder builder)
```

Comment:
I think the rest of the file is using 4 space indentation for parameters, so left shift.